tests/test_serialize.py

```diff
@@ -177,22 +177,8 @@ def no_args(not_self: Any, value: Any, nxt: Any, info: Any) -> Any:  # pragma: n
                 ...
 
 
-def test_invalid_signature_bad_wrap_signature() -> None:
-    with pytest.raises(TypeError, match='Invalid signature for wrap serializer'):
-
-        class _(BaseModel):
-            x: int
-
-            # note that type checkers won't pick this one up
-            # we could fix it but it would require some fiddling with the self argument in the
-            # callable protocols
-            @serializer('x', mode='wrap')
-            def no_args(self, value: Any, info: Any) -> Any:  # pragma: no cover
-                ...
```

Comment:
Note that we can't catch this at runtime or statically anymore because there's no way to distinguish it from `(value, handler)` unless we want to (1) introspect types or (2) force some of these to be keyword arguments (but then we have to construct kwargs in rust).